internal/injector/builtin/yaml/api/vault.yml

```diff
@@ -0,0 +1,34 @@
+# Unless explicitly stated otherwise all files in this repository are licensed
+# under the Apache License Version 2.0.
+# This product includes software developed at Datadog (https://www.datadoghq.com/).
+# Copyright 2024-present Datadog, Inc.
+%YAML 1.1
+---
```

Comment:
We have decided to no longer put the YAML version directive because it messes with Goland's schema validation (among other reasons). Also, now you can bind it to the schema document:

```suggestion
---
# yaml-language-server: $schema=../../../../../docs/static/schema.json
```